e2e/slider/slider.e2e-spec.ts

```diff
@@ -0,0 +1,374 @@
+import {SliderPO} from '@agnos-ui/page-objects';
+import {SliderDemoPO} from '../demo-po/slider.po';
+import {getTest, expect} from '../fixture';
+
+const defaultExpectedState: {[key: string]: string | null} = {
+	value: '60',
+	min: '0',
+	max: '100',
+	text: '60',
+	readonly: null,
+	disabled: null,
+	style: 'left: 60%; top: 0%;',
+};
+
+const defaultExpectedHandleState: {[key: string]: string | null}[] = [
+	{
+		value: '10',
+		min: '0',
+		max: '100',
+		text: '10',
+		readonly: null,
+		disabled: null,
+		style: 'left: 10%; top: 0%;',
+	},
+	{
+		value: '40',
+		min: '0',
+		max: '100',
+		text: '40',
+		readonly: null,
+		disabled: null,
+		style: 'left: 40%; top: 0%;',
+	},
+];
+
+const defaultExpectedHandleStateVertical: {[key: string]: string | null}[] = [
+	{
+		value: '10',
+		min: '0',
+		max: '100',
+		text: '10',
+		readonly: null,
+		disabled: null,
+		style: 'left: 0%; top: 90%;',
+	},
+	{
+		value: '40',
+		min: '0',
+		max: '100',
+		text: '40',
+		readonly: null,
+		disabled: null,
+		style: 'left: 0%; top: 60%;',
+	},
+];
+
+const test = getTest();
+test.describe(`Slider tests`, () => {
+	test.describe(`Basic slider`, () => {
+		test(`should snap the handle to correct tick on the slider click event`, async ({page}, testInfo) => {
+			if (testInfo.project.name.startsWith('react:') || testInfo.project.name.startsWith('svelte:')) {
```

Comment:
Instead of repeating this condition in each test, you can use `test.skip` in `describe`, as documented here:
https://playwright.dev/docs/test-annotations#conditionally-skip-a-group-of-tests

Reply:
I have actually done it in this commit: https://github.com/AmadeusITGroup/AgnosUI/commit/cf480e7cee5b1cc812fa50a8febe14c39294bf93